app/lib/conferences/eventCall.js

```diff
@@ -157,6 +158,53 @@ export const getAllEvents = async (eid) => {
   return res;
 };
 
+export const getSponsorsDetails = async (eid,auth) => {
+  const headers = {
+    Accept: "application/vnd.api+json",
+    Authorization: `JWT ${auth}`,
+  };
+  const res = await axios.get(`${eventUrl}/v1/events/${eid}/sponsors`, {
+    headers: headers,
+  });
+  return res;
+}
```

Comment:
We are using the same call on the EventDisplay page, and this would result in an error. Since the Event Display page does not requires the user to be logged in.

Also, if possible please reduce introducing new `axios` calls, we are trying to move all the API calls, to native `fetch`

Reply:
I have added a new event details call similar to getEventDeatils but with the auth because to fetch the details of draft event we will need the auth.